android/fastlane/README.md

```diff
@@ -15,47 +15,21 @@ Install _fastlane_ using
 or alternatively using `brew cask install fastlane`
 
 # Available Actions
-## Android
-### android keystore
+### keystore
```

Comment:
If you're motivated, you could change this to h2 :)

Reply:
Docs will follow. This is only my scratch pad :)